web/edit/server/index.php

```diff
@@ -1045,6 +1045,19 @@
         }
     }
 
+    // Change API_V2
```

Comment:
Current javascript hides the ip box when disabling legacy api. This should not happen.